service/src/test/java/greencity/service/UserServiceImplTest.java

```diff
@@ -84,12 +84,7 @@
 import org.springframework.test.util.ReflectionTestUtils;
 import java.sql.Timestamp;
 import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
no wildcard imports

Reply:
In intellij you could go to settings -> editor -> code style -> java

make sure Use single class import is enabled
in class count to use import with '\*' and names count to use static import with '\*'  set some bit number (e.g 999) and intellij idea would import each class separately.

Reply:
Thank you for advice to configure settings!